tests/system/fixtures/__init__.py

```diff
@@ -93,3 +93,20 @@ def docker_ipv6_network_fixture():
     yield
     for agent in agents:
         shakedown.run_command_on_agent(agent, f"sudo docker network rm mesos-docker-ipv6-test")
+
+
+@pytest.fixture(autouse=True, scope='session')
+def archive_sandboxes():
```

Comment:
This is really nice! I think we should add it to Shakedown, though. What do you think?